README.md

```diff
@@ -571,10 +571,12 @@ colors, in addition to passing the levels to the Logger itself, you must make
 winston aware of them:
 
 ``` js
-winston.addColors(myCustomLevels.colors);
```

Comment:
I'm not sure why I was thinking that was gone.  I'll have to fix up several places to use `winston.addColors` instead of adding them to the colorizer.  I'll take a look this evening.